rockcraft.yaml

```diff
@@ -0,0 +1,131 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+# https://github.com/apache/superset/blob/master/Dockerfile
+name: charmed-superset-rock
+base: ubuntu@22.04
+version: 2.1.0-22.04-edge
+summary: Charmed Superset ROCK OCI
+description: |
+  Superset is a business intelligence tool that is fast, lightweight,
+  intuitive, and loaded with options that make it easy for users
+  to explore and visualize their data.
+license: Apache-2.0
+
+services:
+  superset-ui:
+    override: replace
+    summary: "superset-ui service"
+    startup: disabled
+    command: "/app/k8s/k8s-bootstrap.sh"
+    environment:
+      CHARM_FUNCTION: "app-gunicorn"
+      SUPERSET_SECRET_KEY: "supersetR0cks!"
+      ADMIN_PASSWORD: "admin"
+      SUPERSET_LOAD_EXAMPLES: True
+
+platforms:
+  amd64:
+
+parts:
+  backend:
+    plugin: python
+    source: https://downloads.apache.org/superset/2.1.0/apache-superset-2.1.0-source.tar.gz  # yamllint disable-line
+    build-packages:
+      - build-essential
+      - wget
+    stage-packages:
+      - python3.10-venv
+    build-environment:
+      - APP_HOME: "/app"
+      - SOURCE: "https://downloads.apache.org/superset"
+      - VERSION: "2.1.0"
+      - DIST_PACKAGES: "usr/local/lib/python3.10/dist-packages/"
+    override-build: |
+      craftctl default
+      # make home directory
+      mkdir -p ${APP_HOME}
+
+      # Download and unpack tar
+      wget "${SOURCE}/${VERSION}/apache-superset-${VERSION}-source.tar.gz" \
+        -O superset.tar.gz
+      tar -zxvf superset.tar.gz -C ${APP_HOME} --strip-components 1
+      rm -rf superset.tar.gz
```

Comment:
Adding a checksum or sha check would make this safer

Reply:
+1

Reply:
Added a sha check, please take another look.